packages/front/src/lib/resources/auth/auth.service.test.js

```diff
@@ -390,5 +390,57 @@ describe("authService", () => {
         });
     });
 
-    // TODO redirectToLogin
+    describe("redirectToLogin", () => {
+        it("saves stringified object", () => {
```

Comment:
pas fan des multiples `expect()` dans un test et jusqu'ici on ne faisait pas vraiment ça, non ?

Reply:
Ben j'ai l'impression qu'on ne le faisait pas parce que la grande majorité du temps ça ne s'y prête pas. Là c'est compliqué de matcher l'objet d'un coup et ça fait très bizarre de faire des `it()` différents pour chacune des propriétés

Reply:
On a pas défini de norme donc ça me va mais j'aurai quand même scindé en plusieurs it, au moins 1 pour le nom du champ et l'autre pour le contenu. Encore une fois c'est personnel donc ça me va

Reply:
J'ai enlevé `JSON.parse` et `JSON.stringify` que j'avais partout donc ç'a simplifié ce test ;

Reply:
ok; tu peux merge :)